### PR TITLE
*: merge experimental-feature.md to basic-features.md

### PR DESCRIPTION
**Pull request by TomShawn**
### What is changed, added or deleted? (Required)

Merge `experimental-features.md` to `basic-features.md` because they have much in common and need extra efforts for update. Some features in `experimental-features.md` are missing in `basic-features.md`, and these features are added to `basic-features.md`.

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions (in Chinese)](https://github.com/pingcap/docs-cn/blob/master/CONTRIBUTING.md#版本选择指南).

- [x] master (the latest development version)
- [x] v6.6 (TiDB 6.6 versions)
- [x] v6.5 (TiDB 6.5 versions)
- [ ] v6.4 (TiDB 6.4 versions)
- [ ] v6.1 (TiDB 6.1 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

File: basic-features.md [hunk 1]
```diff
@@ -38,8 +38,10 @@ aliases: ['/docs-cn/dev/basic-features/']
 | ---------------------------------------------------------------------- | --- | --- | --- | --- | --- | --- | :------: | :------: | :------: | :------: | :------: | :------: |
 | [表达式索引](/sql-statements/sql-statement-create-index.md#表达式索引) [^2] | Y | 实验特性 | 实验特性 | 实验特性  | 实验特性  | 实验特性  | 实验特性 | 实验特性 | 实验特性 | 实验特性 | 实验特性 | 实验特性 |
 | [列式存储 (TiFlash)](/tiflash/tiflash-overview.md)                     |  Y   |  Y   |  Y   |  Y   |  Y    |  Y    |    Y     |    Y     |    Y     |    Y     |    Y     |    Y     |
+| [物化列式存储的查询结果](/tiflash/tiflash-results-materialization.md) | 实验特性 | N | N | N | N | N | N | N | N | N | N | N |
```
Comment (TomShawn) [inline basic-features.md:41]: SQL

File: basic-features.md [hunk 2]
```diff
@@ -38,8 +38,10 @@ aliases: ['/docs-cn/dev/basic-features/']
...
 | [RocksDB 引擎](/storage-engine/rocksdb-overview.md)                    |  Y   |  Y   |  Y   |  Y    |  Y    |  Y    |    Y     |    Y     |    Y     |    Y     |    Y     |    Y     |
 | [Titan 插件](/storage-engine/titan-overview.md)                        |  Y   |  Y   |  Y   |  Y    |  Y    |  Y    |    Y     |    Y     |    Y     |    Y     |    Y     |    Y     |
```
Comment (TomShawn) [inline basic-features.md:43]: 文档分类参照 MySQL？

File: basic-features.md [hunk 3]
```diff
@@ -82,6 +86,9 @@ aliases: ['/docs-cn/dev/basic-features/']
 | [MPP 执行引擎](/explain-mpp.md)                                     |    Y    |    Y    |    Y    |    Y    |    Y    |    Y    |    Y     |    Y     |    Y     |    Y     |    Y     | N        |
 | [索引合并](/explain-index-merge.md)                                 |    Y    |    Y    |    Y    |    Y    |    Y    |    Y    |    Y     | 实验特性 | 实验特性 | 实验特性 | 实验特性 | 实验特性 |
 | [基于 SQL 的数据放置规则](/placement-rules-in-sql.md)                |    Y    |    Y    |    Y    |    Y   |    Y    |    Y    | 实验特性 | 实验特性 |    N     |    N     |    N     | N        |
+| [使用 FastScan 加速 OLAP 场景下的查询](/develop/dev-guide-use-fastscan.md) | 实验特性 | 实验特性 | 实验特性 | 实验特性 | N | N | N | N | N | N | N | N |
```
Comment (TomShawn) [inline basic-features.md:89]: 放 TiFlash 下面

File: basic-features.md [hunk 4]
```diff
@@ -82,6 +86,9 @@ aliases: ['/docs-cn/dev/basic-features/']
...
+| [Cascades Planner](/system-variables.md#tidb_enable_cascades_planner) | 实验特性 | 实验特性 | 实验特性 | 实验特性 | 实验特性 | 实验特性 | 实验特性 | 实验特性 | 实验特性 | 实验特性 | 实验特性 | 实验特性 |
```
Comment (TomShawn) [inline basic-features.md:90]: 删掉，用户不需要关注

File: basic-features.md [hunk 5]
```diff
@@ -193,9 +205,12 @@ aliases: ['/docs-cn/dev/basic-features/']
 | [`SET CONFIG`](/dynamic-config.md)                                        | Y  | Y  | Y | Y | Y | 实验特性 | 实验特性 | 实验特性 | 实验特性 | 实验特性 | 实验特性 | 实验特性 | 实验特性 |
 | [DM WebUI](/dm/dm-webui-guide.md)                                         | 实验特  | 实验特  | 实验特性 | 实验特性 | 实验特性 | 实验特性 |    N     |    N     |    N     |    N     |    N     | N        |
 | [前台限流](/tikv-configuration-file.md#前台限流)                            | Y | Y | Y | Y | 实验特性 | 实验特性 |    N     |    N     |    N     |    N     |    N     | N        |
+| [后台限流](/tikv-configuration-file.md#后台限流)                             | 实验特性 | 实验特性 | 实验特性 | 实验特性 | N | N | N | N | N | N | N | N |
```
Comment (TomShawn) [inline basic-features.md:208]: 如果有「存储」，放存储下更好

File: basic-features.md [hunk 6]
```diff
@@ -193,9 +205,12 @@ aliases: ['/docs-cn/dev/basic-features/']
...
 | [基于 EBS 的备份和恢复](https://docs.pingcap.com/zh/tidb-in-kubernetes/v1.4/volume-snapshot-backup-restore)                        |    Y    |    Y    |    N     |    N     |    N    |    N     |    N     |    N     |    N     |    N     |    N     |  N        |
 | [PITR](/br/br-pitr-guide.md)   |    Y    |    Y    |    Y     |    Y     |    N    |    N     |    N     |    N     |    N     |    N     |    N     |  N        |
 | [全局内存控制](/configure-memory-usage.md#如何配置-tidb-server-实例使用内存的阈值)   |    Y    |    实验特性    |    N     |    N     |    N    |    N     |    N     |    N     |    N     |    N     |    N     |  N        |
+| [RawKV 跨集群复制](/tikv-configuration-file.md#api-version-从-v610-版本开始引入) | 实验特性 | 实验特性 | 实验特性 | 实验特性 | N | N | N | N | N | N | N | N |
+| [Green GC](/system-variables.md#tidb_gc_scan_lock_mode-从-v50-版本开始引入) | 实验特性 | 实验特性 | 实验特性 | 实验特性 | 实验特性 | 实验特性 | 实验特性 | 实验特性 | 实验特性 | 实验特性 | 实验特性 | N |
```
Comment (TomShawn) [inline basic-features.md:213]: 最好单独开一个部分

File: basic-features.md [hunk 7]
```diff
@@ -1,7 +1,7 @@
 ---
 title: TiDB 基本功能
 summary: 了解 TiDB 的基本功能。
-aliases: ['/docs-cn/dev/basic-features/']
+aliases: ['/docs-cn/dev/basic-features/','/docs-cn/dev/experimental-features-4.0/','/zh/tidb/dev/experimental-features-4.0/', '/zh/tidb/dev/experimental-features']
```
Comment (TomShawn) [inline basic-features.md:4]:
```suggestion
aliases: ['/docs-cn/dev/basic-features/','/docs-cn/dev/experimental-features-4.0/','/zh/tidb/dev/experimental-features-4.0/','/zh/tidb/dev/experimental-features']
```

File: basic-features.md [hunk 8]
```diff
@@ -1,7 +1,7 @@
 ---
 title: TiDB 基本功能
 summary: 了解 TiDB 的基本功能。
-aliases: ['/docs-cn/dev/basic-features/']
+aliases: ['/docs-cn/dev/basic-features/','/docs-cn/dev/experimental-features-4.0/','/zh/tidb/dev/experimental-features-4.0/','/zh/tidb/dev/experimental-features']
 ---
 
 # TiDB 基本功能
```
Comment (qiancai) [inline basic-features.md:7]: 因为实验特性文档移除了，所以可以考虑在这篇文档里说明下实验特性的特殊之处，比如在正文第一句话后，加一个说明。 

警告：

针对本文中列出的实验性功能，请注意使用场景限制。该功能会在未事先通知的情况下发生变化或删除。语法和实现可能会在 GA 前发生变化。如果遇到问题，请在 GitHub 上提交 [issue](https://github.com/pingcap/tidb/issues) 反馈。

**Review by qiancai — APPROVED** [review]
LGTM

**Pull request merged**